README.md

```diff
@@ -110,6 +110,8 @@ dependencies {
 }
 ```
 
+Note: dslGenerateXml fails if ws-cleanup. This is a known issue tracked by [#74](https://github.com/heremaps/gradle-jenkins-jobdsl-plugin/issues/74).
```

Comment:
The first sentence is missing the end, please change it to: "...if `org.jenkins-ci.plugins:ws-cleanup` is added to the `jenkinsPlugins` configuration."

Reply:
Done, I think my brain had an interrupt while writing that sentence.